admin.py

```diff
@@ -381,7 +381,10 @@ def post(self, path, feature_id=None):
       common.handle_401(self.request, self.response, Exception)
       return
 
+    spec_repo_url = self.__FullQualifyLink('spec_repo_url')
```

Comment:
I'm confused.  Do we have a spec repo URL *AND* a spec link (URL)?  That seems redundant.

Reply:
+1

Reply:
I'll remove this from this CL.  Chris, I think that this had been "WICG repo" in an earlier version and we renamed it to be more generic, but I could be remembering that wrong.  In any event, if you do want some kind of incubation repo link, I can add it back in a future CL.